ui/console/src/main/scripts/plugins/metrics/plugins/metrics/html/response-time.html

```diff
@@ -57,7 +60,7 @@ <h3 class="pull-left">Metrics</h3>
       </div>
     </div>
 
-    <div class="panel panel-default hk-graph">
+    <div class="panel panel-default hk-graph" ng-if="vm.average >= 0">
```

Comment:
I think we want to use ng-show/ng-hide instead of ng-if as it removes/adds DOM which is expensive.